azure-communication-ui/azure-communication-ui-demo-app/build.gradle

```diff
@@ -90,12 +93,20 @@ dependencies {
     implementation project(path: ':azure-communication-ui')
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     implementation "androidx.core:core-ktx:$androidx_core_version"
-    implementation "androidx.appcompat:appcompat:$appcompat_version"
+    implementation ("androidx.appcompat:appcompat:$appcompat_version") {
+        exclude group: "androidx.activity", module: "activity"
+    }
     implementation 'androidx.activity:activity-ktx:1.3.1'
     implementation 'com.google.android.material:material:1.4.0'
     implementation "androidx.constraintlayout:constraintlayout:$constraint_layout"
     implementation 'com.github.kittinunf.fuel:fuel:2.3.1'
-    implementation "com.azure.android:azure-communication-calling:$calling_sdk_version"
+    implementation "com.azure.android:azure-core:$azure_core_version"
+    implementation "com.google.code.gson:gson:$gson_version"
+    implementation ("com.azure.android:azure-communication-calling:$calling_sdk_version") {
+        exclude group: "com.azure.android", module: "azure-core"
```

Comment:
Are these excludes followed by re-imports necessary? E.g. GSON.

Reply:
Also, risk of breakages?

Reply:
Also, side question? Why do we bring azure-communication-calling into demo, shouldn't it come transitively via azure-communication-ui?

Reply:
The gson version update is a dependency that CallingSDK depends on.  The reason I wanted to update to newest GSON is to fix a [security warning](https://skype.visualstudio.com/SCC/_componentGovernance/client_calling_spool-native/alert/6140890?typeId=5727151) about the older version of GSON.  Technically it's not our responsibility but it looked like low hanging fruit.  I can remove if you want.

we bring azure-communication-calling into demo because the Error Handler is a CallingSDK construct

Reply:
I think we should not make exclusion of what CallingSDK team decided to include. It should be up to them to make this decision.

Reply:
@pavelprystinka azure-communication-ui library module line 84 is already including newer version of azure core lib, should that be removed?

Reply:
@pavelprystinka  ok nevermind, I found a solution